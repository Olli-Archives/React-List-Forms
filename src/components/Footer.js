import React from 'react';
import PropTypes from 'prop-types';


export default function Footer({ name }){
  return <footer>{name}</footer>;
}


Footer.propTypes = {
  name: PropTypes.string.isRequired
}
;
