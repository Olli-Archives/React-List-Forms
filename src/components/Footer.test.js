import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

const myName = 'Olli Nieminen';

describe('Footer component', ()=>{
  it('renders name to footer', ()=>{
    const wrapper = shallow(<Footer name={myName}/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
