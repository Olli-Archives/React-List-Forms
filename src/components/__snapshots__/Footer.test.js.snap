// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders name to footer 1`] = `
<footer>
  Olli Nieminen
</footer>
`;
